lib/chunky_png/chunk.rb

```diff
@@ -368,13 +368,35 @@ def content
     # a translation of the keyword name. Finally, it supports bot compressed
     # and uncompressed values.
     #
-    # @todo This chunk is currently not implemented, but merely read and
-    #   written back intact.
+    # http://www.libpng.org/pub/png/spec/1.2/PNG-Chunks.html#C.iTXt
     #
     # @see ChunkyPNG::Chunk::Text
     # @see ChunkyPNG::Chunk::CompressedText
     class InternationalText < Generic
-      # TODO
+      attr_accessor :keyword, :text, :language_tag, :translated_keyword, :compressed, :compression
+
+      def initialize(keyword, text, language_tag = '', translated_keyword = '', compressed = ChunkyPNG::UNCOMPRESSED_CONTENT, compression = ChunkyPNG::COMPRESSION_DEFAULT)
+        super('iTXt')
+        @keyword =keyword
+        @text = text
+        @language_tag = language_tag
+        @translated_keyword = translated_keyword
+        @compressed = compressed
+        @compression = compression
+      end
+
+      def self.read(type, content)
+        keyword, compressed, compression, language_tag, translated_keyword, text_field = content.unpack('Z*CCZ*Z*a*')
```

Comment:
Hello, I will fix file permission and space. Could you please give me hint about encoding, according to PNG specs: "The translated keyword and text both use the UTF-8 encoding of the Unicode character set", what should I use to encode UTF-8 string in content method?